Examples/LocationManager/Common/MapView.swift

```diff
@@ -55,10 +55,16 @@ public struct MapView: ViewRepresentable {
       mapView.setRegion(region.asMKCoordinateRegion, animated: true)
     }
 
-    mapView.removeAnnotations(mapView.annotations)
-    mapView.addAnnotations(
-      self.pointsOfInterest.map(PointOfInterestAnnotation.init(pointOfInterest:))
-    )
+    let currentlyDisplayedPOIs = mapView.annotations.compactMap { $0 as? PointOfInterestAnnotation }.map(\.pointOfInterest)
```

Comment:
Can we make this change so that this example continues building for iOS 13.3, which sadly doesn't have key path functions.

```suggestion
    let currentlyDisplayedPOIs = mapView.annotations.compactMap { $0 as? PointOfInterestAnnotation }.map { $0.pointOfInterest }
```